src/DependencyInjection/Factory/BlockAdminFactory.php

```diff
@@ -70,9 +73,11 @@ public function create(array $config, ContainerBuilder $container, XmlFileLoader
 
         $bundles = $container->getParameter('kernel.bundles');
 
-        if (true === $config['enable_menu']
-            || ('auto' === $config['enable_menu'] && isset($bundles['CmfMenuBundle']))
-        ) {
+        $message = 'CmfMenuBundle integration was explicitely enabled, but the bundle is not available';
+        if (class_exists(CmfMenuBundle::class)) {
+            $message .= ' (did you forget to register the bundle in the AppKernel?)';
+        }
+        if ($this->isConfigEnabledAuto($container, $config['enable_menu'], 'CmfMenuBundle', $message.'.')) {
```

Comment:
I don't agree with this. Please note that the exception is thrown when `enable_menu` is set to `true` (so not `auto`), while the bundle is not available. I think it's important to tell people that it's explicitly set to true.